stubs/dateparser/@tests/requirements-stubtest.txt

```diff
@@ -0,0 +1,2 @@
+fasttext
```

Comment:
It would be better to just specify the `extras` in `METADATA.toml`, similar to https://github.com/python/typeshed/blob/3c9d1bd942c9ae754156601ec67f160a192f271f/stubs/requests/METADATA.toml#L6